packages/client/src/app/root/components/MainWindow.tsx

```
@@ -1,57 +1,34 @@
-          layers={layers}
-          componentWithGrid={componentWithGrid}
-        />
-      ))}
+      {
+        renderedComponents
+          .map(({ uiComponent, gridConfig }) => (
+            <div
+              style={{
+                gridArea: `${gridConfig.rowStart} / ${gridConfig.colStart} / ${gridConfig.rowEnd} / ${gridConfig.colEnd}`,
+              }}
+            >
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)